conversation_reconstruction_dataflow/ingest_revisions/dataflow_main.py

```diff
@@ -166,19 +197,20 @@ def process(self, element):
                       help='If you want to run the input dumps in batch, pick a batch number to run')
   parser.add_argument('--input',
                       dest='input',
-                      default='gs://wikidetox-viz-dataflow/input_lists/7z_file_list_stuck',
+                      default='gs://wikidetox-viz-dataflow/input_lists/7z_file_list_reingest',
                       help='Input file to process.')
   # Destination BigQuery Table
-  schema = 'sha1:STRING,user_id:STRING,format:STRING,user_text:STRING,timestamp:STRING,text:STRING,page_title:STRING,model:STRING,page_namespace:STRING,page_id:STRING,rev_id:STRING,comment:STRING, user_ip:STRING, truncated:BOOLEAN,records_count:INTEGER,record_index:INTEGER'
-  parser.add_argument('--table',
-                      dest='table',
-                      default='wikidetox-viz:wikidetox_conversations.ingested_conversations_stuck',
-                      help='Output table to write results to.')
+  schema = 'sha1:STRING,user_id:STRING,format:STRING,user_text:STRING,timestamp:STRING,text:STRING,page_title:STRING,model:STRING,page_namespace:STRING,page_id:STRING,rev_id:INTEGER,comment:STRING, user_ip:STRING, truncated:BOOLEAN,records_count:INTEGER,record_index:INTEGER'
```

Comment:
If this is for BigQuery, then Timestamp should not be a string. It should be a date, no?

Reply:
Yes, but this is also necessary since we are writing to file now.

Reply:
Not sure I understand, when you write to a file, schema is not used is it?

Reply:
oops sorry I meant this is also **not** necessary since we are writing to file now.